src/Microsoft.TemplateEngine.Cli/CliTemplateEngineHost.cs

```diff
@@ -117,20 +117,58 @@ public bool OnPotentiallyDestructiveChangesDetected(IReadOnlyList<IFileChange> c
             return false;
         }
 
-        public bool OnConfirmPartialMatch(string name)
+        #region Obsolete
+        [Obsolete]
+        void ITemplateEngineHost.LogMessage(string message)
+        {
+            //do nothing
```

Comment:
This host is internal, and none of these methods are used now. They are needed only for interface implementation.

Reply:
In theory when obsoleting API... We would like API to continue working if possible, for example lets say that some Component likes to log things... now in .NET 6 it won't work anymore... Ideally it would continue to work(by just calling `logger.LogInformation(message);`) so it has 1 year until .NET 7 to stop depending on this obsolete API...